javascript-carousel/main.js

```diff
@@ -0,0 +1,63 @@
+var $img = document.querySelector('img');
+var $circles = document.querySelectorAll('i.circle');
+var $rightArrow = document.querySelector('i[id="right"]');
+var $leftArrow = document.querySelector('i[id="left"]');
+var intervalTime = setInterval(rightPic, 3000);
+var $images = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var imageIndex = 0;
+
+for (var i = 0; i < $circles.length; i++) {
+  $circles[i].addEventListener('click', goToIndex);
+  $circles[i].circleIndex = i;
```

Comment:
You should use a `data-` attribute rather than making custom properties on DOM elements.